packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -179,3 +177,99 @@ def delete_worker(
         return SyftSuccess(
             message=f"Worker with id: {worker_id} deleted successfully from pool: {worker_pool.name}"
         )
+
+    @service_method(
+        path="worker_pool.worker_logs",
+        name="worker_logs",
+        roles=DATA_OWNER_ROLE_LEVEL,
+    )
+    def worker_logs(
+        self,
+        context: AuthedServiceContext,
+        worker_pool_id: UID,
+        worker_id: UID,
+        raw: bool = False,
```

Comment:
@rasswanth-s added this argument. By default the endpoint returns `str` logs (`.decode()`). Users can opt to get the raw `bytes` logs if they want.